### ISSUE TITLE
mvp - fix heading levels

### ISSUE THREAD

**Issue by joelanman** [alphagov/datagovuk_find#1648]
There should only be 1 h1 ideally, and headings should descend without skipping levels:

good: h1, h2, h2, h3, h2, h3
bad: h1, h1, h3, h2, h4

- [ ] Home – Collections should be h2 not h1 

- [ ] Roadmap – the headings in the Now and Next sections should be h4 not h5
